tests/data/trips_mod_columns.csv

```diff
@@ -1,5 +1,5 @@
-id;user_id;started_at;finished_at;orig_stp;dest_stp
-0;1;2015-11-27T08:00:00Z;2015-11-27T08:15:00Z;2;5
-1;1;2015-11-27T08:20:22Z;2015-11-27T08:35:22Z;2;5
-2;1;2015-11-27T12:39:28Z;2015-11-27T12:55:28Z;2;5
-3;1;2015-11-27T13:57:20Z;2015-11-27T14:13:20Z;2;5
\ No newline at end of file
+id;user_id;started_at;finished_at;orig_stp;dest_stp;geom
```

Comment:
I left the geom column here in order to test whether the dataframes are equal (except for this column) - Tested in `test_file.py`. Apart from this one and the `geolife_long/trips.csv`, I changed the test data back to dataframes, since trips tables now support either dataframes or geodataframes.

Reply:
Ah ok. So to have some test data with and some without geometry?

Reply:
Yes, as said above, in `test_file.py` some tests read csv files with geometry and others without, and the two are compared as well for consistency